contrib/mpd/files/mpd

```diff
@@ -0,0 +1,9 @@
+# MPD service
+
+type            = process
+command         = /usr/bin/mpd --no-daemon
+before          = network.target
+depends-on      = pre-network.target
+restart         = true
+smooth-recovery = true
+
```

Comment:
there's an extra trailing newline here, also don't need restart=true, and don't need to align service file = anymore